packages/breadcrumb/src/js/breadcrumb.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+/**
+ * Breadcrumb module
+ * @file Handles interaction behaviour for the breadcrumb.
+ */
+
+class Breadcrumb {
+  constructor(el) {
+    this.init(el);
+  }
+
+  init(el) {
+    const breadcrumbs = el.querySelectorAll(".uq-breadcrumb__item");
+
+    // truncate if more than 4 options
+    if (breadcrumbs.length > 4) {
```

Comment:
@uqnm You could try changing this condition to be `<= 3` and return early. Something like:
```
if (breadcrumbs.length <= 3) {
  return;
}

The rest of the code can then sit here....
```